fuzz/croaring_fuzzer.c

```diff
@@ -88,4 +89,17 @@ int LLVMFuzzerTestOneInput(const char *data, size_t size) {
     } else {
         return bitmap64(data + 1, size - 1);
     }
+=======
```

Comment:
Looks like some git merge conflict markers got committed here.

Reply:
Ah. Damn it. Fixed.